app/models/invenio_rdm_record_converter.rb

```diff
@@ -375,6 +375,15 @@ def format_publication_date(publication_date)
     normalize_date(publication_date)
   end
 
+  def format_dates(dates)
```

Comment:
This seemed like the easiest way to add the logic we needed. There might be a way to make it a one-liner with the `{}` block syntax, but if there is it will be very, very ugly. This may not be super good looking, but it gets the point across.